source/core/smarty/plugins/function.oxscript.php

```diff
@@ -197,6 +197,12 @@ function _oxscript_execute( $aScript, $sWidget, $blAjaxRequest )
     return $sOutput;
 }
 
+function _oxscript_sanitize($sScripts)
+{
+    $sScriptTokenSanitized = strtr($sScripts,['"' => '\"',"\r" =>'',"\n"=>'\n']);
```

Comment:
Since oxid still supports PHP 5.3 [1] you need to use the old array() syntax instead of the short array syntax introduced in PHP 5.4.
Just commit the change to your branch *keywan-ghadami-oxid:fix-oxscript-newline* and it will be automatically applied to this pull request.

[1] http://wiki.oxidforge.org/Downloads/4.9.6_5.2.6